src/test/java/de/tum/cit/aet/artemis/iris/IrisProgrammingExerciseChatSessionIntegrationTest.java

```diff
@@ -196,6 +201,125 @@ void testDeleteExerciseWithIrisMessagesWithJsonMessageContent() throws Exception
         assertThat(irisMessageRepository.findAllBySessionId(irisSession.getId())).isEmpty();
     }
 
+    @Test
+    @WithMockUser(username = TEST_PREFIX + "student1", roles = "USER")
+    void testRequestMessageFromIris_withUncommittedFiles() {
+        // Arrange
+        var session = createSessionForUser("student1");
+        var message = createDefaultMockTextMessage(session);
+        irisMessageService.saveMessage(message, session, IrisMessageSender.USER);
+
+        var uncommittedFiles = Map.of("src/Main.java", "public class Main { /* uncommitted changes */ }", "src/Utils.java", "public class Utils { /* new file */ }");
```

Comment:
I think that `var` is overused in this test file in general and makes the tests harder to maintain than it would be necessary

```suggestion
        Map<String, String> uncommittedFiles = Map.of("src/Main.java", "public class Main { /* uncommitted changes */ }", "src/Utils.java", "public class Utils { /* new file */ }");
```